### PR TITLE
[add] `README.md`, improved demo functionality, more comments

### PR DESCRIPTION
**Pull request by sassbalint**
`emDummy` improved with useful features from `emStart`.
Review and merge, please.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+ * takes the value of the `form` field;
+ * create a new field called `star` which will contain the value of the `form` field together with an added asterisk on both sides.
+
+E.g. if the `form` field is `kutya` the `star` field will be `*kutya*`.
+
+It is demonstrated that the order of the columns does _not_ affect the operation of `xtsv`.
+
+For a bit more advanced example see: [`emdummy`](https://github.com/nytud/emdummy).
+
+## python package creation
```
Comment (dlazesz) [inline README.md]: Please write Python with capital letter in text. (Other places as well.)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+3. the package is installed in `venv`;
+4. the package is tested (see __testing__).
+
+The python package can be installed anywhere by direct path:
+
+```bash
+pip install path/to/emdummy-*-py3-none-any.whl
+```
+
+## testing
```
Comment (dlazesz) [inline README.md]: Title should be initial uppercase: Testing

File: emdummy/__main__.py [hunk 3]
```diff
@@ -26,13 +26,32 @@ def main():
...
     # from emdummy import EmDummy
-    em_dummy = ('emdummy', 'EmDummy', 'EXAMPLE (The friendly name of EmDummy used in REST API form)',
-                ('Params', 'goes', 'here'), {'source_fields': {'form'},  # Source field names
-                                             'target_fields': ['star']})  # Target field names
-    tools = [(em_dummy, ('dummy', 'dummy-tagger', 'emDummy'))]
+    em_dummy = (
+        'emdummy', # module name
+        'EmDummy', # class
+        'EmDummy, just add stars to `form`', # friendly name used in REST API form
+        (), # params (currently none)
```
Comment (dlazesz) [inline emdummy/__main__.py]: The word "params" is inaccurate as one must distinguish args from kwargs. In line 33 one can suply args and in line 34 a dictionary of kwargs (currently `source_fields` and `target_fields`).

Please correct the comment to include the above.

**Review by dlazesz — CHANGES_REQUESTED** [review]
Please consider the requested minor modifications.

**Comment by sassbalint** [comment]
Done. :)

**Review by dlazesz — APPROVED** [review]

**Comment by dlazesz** [comment]
Thank you!

**Pull request merged**